app/ios/iosdevice.h

```diff
@@ -0,0 +1,55 @@
+#ifndef QIDEVICE_H
+#define QIDEVICE_H
+
+#include <QObject>
+#include <QVariantMap>
+
+/// Device information provider
+
+class QIDevice : public QObject
```

Comment:
the name suggests it is part of QT framework, maybe some more "Input"-app prefix?